extensions/resteasy-server-common/spi/src/main/java/io/quarkus/resteasy/server/common/spi/FriendlyJaxRsAnnotationPrefixBuildItem.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.resteasy.server.common.spi;
+
+import org.jboss.jandex.DotName;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public final class FriendlyJaxRsAnnotationPrefixBuildItem extends MultiBuildItem {
```

Comment:
This needs proper Javadoc. Not entirely sure about the name.

Reply:
I added some